tests/Unit/Controllers/DriftControllerTest.php

```
@@ -620,4 +620,41 @@ public function testReviewChangeApproveWithNoRelatedItemId(): void
+            $callNum++;
+            if ($callNum === 1) {
+                return ['id' => 5, 'project_id' => 1, 'status' => 'pending', 'proposed_change_json' => '{}'];
+            }
+            if ($callNum === 2) {
+                return ['id' => 1, 'name' => 'Project', 'org_id' => 10];
+            }
+            return false;
+        });
+        $stmt->method('fetchAll')->willReturn([]);
+
+        $db = $this->createMock(\StratFlow\Core\Database::class);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)